app/src/componentLibrary/autocompletes/multipleAutocomplete/multipleAutocomplete.jsx

```diff
@@ -52,6 +52,7 @@ export const MultipleAutocomplete = ({
   handleUnStoredItemCb,
   dataAutomationId,
   existingItemsMap,
+  hideSelected,
```

Comment:
I suppose you can try to update `filteredOptions` calculation for this purpose, line [216](https://github.com/reportportal/service-ui/pull/3644/files#diff-d2a98203eb9b9a2f86edc72601251ccf773b23afe12b241ddf9f7d44f5d5fd47R216)